db/migrations/20220420023434_update_curation_tag_materialized.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * This migration script is for generating curation tags ID list.
+ *
+ */
+const view = 'curation_tag_materialized'
+
+exports.up = async (knex) => {
+  // drop current materialized view
+  await knex.raw(`DROP MATERIALIZED VIEW IF EXISTS ${view}`)
+
+  // recreate new materialized view
+  await knex.raw(`
+    CREATE MATERIALIZED VIEW ${view} AS
+    SELECT t.id, 1/(SUM(t1.sum_read_time)+1) AS uuid, SUM(t1.sum_read_time) AS sum_read_time_top_n
```

Comment:
`RANDOM() AS uuid` ?

Reply:
@tx0c you are correct, I added the randomness back. please check.